docs/flink/flink-getting-started.md

```diff
@@ -70,33 +69,34 @@ Step.3 Start the flink SQL client.
 
 We've created a separate `flink-runtime` module in iceberg project to generate a bundled jar, which could be loaded by flink SQL client directly.
 
-If we want to build the `flink-runtime` bundled jar manually, please just build the `iceberg` project and it will generate the jar under `<iceberg-root-dir>/flink-runtime/build/libs`. Of course, we could also download the `flink-runtime` jar from the [apache official repository](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-flink-runtime/).
+If we want to build the `flink-runtime` bundled jar manually for a specific Flink version, such as v1.15, please just build the `iceberg` project and it will generate the jar under `<iceberg-root-dir>/flink/v1.15/flink-runtime/build/libs`. Of course, we could also download the `flink-runtime-1.15` jar from the [Apache official repository](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-flink-runtime-1.15/).
 
 ```bash
 # HADOOP_HOME is your hadoop root directory after unpack the binary package.
 export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
 
-./bin/sql-client.sh embedded -j <flink-runtime-directory>/iceberg-flink-runtime-xxx.jar shell
+./bin/sql-client.sh embedded -j <flink-runtime-directory>/iceberg-flink-runtime-1.15-xxx.jar shell
 ```
 
-By default, iceberg has included hadoop jars for hadoop catalog. If we want to use hive catalog, we will need to load the hive jars when opening the flink sql client. Fortunately, apache flink has provided a [bundled hive jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-hive-2.3.6_2.11/1.11.0/flink-sql-connector-hive-2.3.6_2.11-1.11.0.jar) for sql client. So we could open the sql client
+By default, iceberg has included hadoop jars for hadoop catalog. If we want to use hive catalog, we will need to load the hive jars when opening the flink sql client. Fortunately, apache flink has provided a [bundled hive jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-hive-2.3.6_2.12/1.15.0/flink-sql-connector-hive-2.3.6_2.12-1.15.0.jar) for sql client. So we could open the sql client
 as the following:
 
 ```bash
 # HADOOP_HOME is your hadoop root directory after unpack the binary package.
 export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
 
 # download Iceberg dependency
-ICEBERG_VERSION=0.11.1
-MAVEN_URL=https://repo1.maven.org/maven2
+ICEBERG_VERSION=0.13.1
+MAVEN_URL=https://repo.maven.apache.org/maven2
 ICEBERG_MAVEN_URL=${MAVEN_URL}/org/apache/iceberg
-ICEBERG_PACKAGE=iceberg-flink-runtime
+ICEBERG_FLINK_VERSION=1.15
```

Comment:
This script won't work because  iceberg 0.13.1 does not support flink 1.15 .

Reply:
@openinx 
Yes, I used Flink 1.14.3 at the beginning, I can be sure that all versions are completely correct, but after I modified the Merge, no one helped me merge it for a long time. I thought it was version upgrade, so I changed it to 1.15.0.

Reply:
I have modified it to flink 1.14.4

Reply:
@openinx updated.